cargo-nextest.hcl

```diff
@@ -0,0 +1,28 @@
+homepage = "https://github.com/nextest-rs/nextest"
+description = "A next-generation test runner for Rust."
+binaries = ["cargo-nextest"]
+strip = 1
+
+platform "darwin" {
+  source = "https://github.com/nextest-rs/nextest/releases/download/cargo-nextest-${version}/cargo-nextest-${version}-universal-apple-darwin.tar.gz"
+}
+
+platform "linux" "amd64" {
```

Comment:
Both these blocks are the same so you could just use `platform liinux` and delete one.